dropboxfs.py

```diff
@@ -169,6 +169,7 @@ def read(self, amt=None):
             return self.r.read(amt)
         else:
             self.close()
+            return ''
```

Comment:
Not sure if this is supposed to work on Python 3, but if it does, you may want to return bytes instead (`b''`).